swap_meet/item.py

```diff
@@ -1,2 +1,40 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=0):
+        if id == None:
+            id = uuid.uuid4().int
+        elif not isinstance(id, int):
+            raise ValueError("id must be an integer")
+        self.id = id
+        self.condition = condition
+        self.age = age
+
+    def get_category(self):
+        """This function returns the category of the item"""
+        return "Item"
+
+    def __str__(self):
+        """This function a string with the object type and id"""
+        return f"An object of type {self.get_category()} with id {self.id}."
+
+    def condition_description(self):
+        """This function returns a string depending on the condition of the item"""
+        if isinstance(self.condition, int) or isinstance(self.condition, float):
```

Comment:
This is probably something that should be checked when the instance is created. If we know we want condition to be an int or a float, we should prevent the instance from being created with this invalid input.